test/sdft.jl

```diff
@@ -81,4 +84,19 @@ dfty_sample = [fft(view(y, (1:n) .+ offset)) for offset in sample_offsets]
         @test_throws "insufficient data" iterate(SDFT(10)(Float64[]))
         @test_throws "previous DFT results not available" collect(ErrorSDFT()(y))
     end
+
+    # Additional coverage
+    @testset "Extra" begin
+        itr = SDFT(n)(y)
+        _, state = iterate(itr)
+        @test ismissing(Base.isdone(itr))
+        @test ismissing(Base.isdone(itr, state))
+        FourierTools.sdft_updatedfthistory!(nothing)
```

Comment:
The codecov threshold is passed, that's good. But I wonder why some lines are still reported as not covered, when these lines of `test/sdft.jl` call them explicitly!